services/QuillLMS/app/controllers/statuses_controller.rb

```diff
@@ -38,7 +38,9 @@ def sidekiq_queue_latency
     queues = Sidekiq::Queue.all
     latency_hash = queues.to_h { |q| [q.name, q.latency] }
 
-    render json: latency_hash
```

Comment:
It might make sense to change the name of the method from sidekiq_queue_latency to `sidekiq_critical_external_latency`

Reply:
It could, but it does technically return the latency for every endpoint?  I suppose we could make the 400 part a parameter of some sort.  Like `/statuses/sidekiq_queue_latency/:queue/:latency` and if `queue` has a latency over `latency` it has status 400, otherwise 200, and it always returns the same body?

Reply:
I was focusing on the response_status and forgot you were sending the latency_hash too.  Please disregard.

I think the parameter passing is a good idea if we end up using this on other queues.